src/models/jwtPermissions/index.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+const { JWTVerifier } = require('@storyous/storyous-jwt');
+const fs = require('fs');
+const path = require('path');
+
+// eslint-disable-next-line import/no-unresolved
+const { permissionHelper, fetch } = require('../../index');
+const { NotSufficientPermissions } = require('./customErrors');
+const { isProduction, isTesting } = require('../../config');
+
+const publicKeys = {};
+let _publicKeyUrl;
+let keyName;
+if (isProduction()) { _publicKeyUrl = 'http://publickey.storyous.com'; keyName = 'publicProduction'; } else
+if (isTesting()) {
```

Comment:
this `else if` looks weird, better to have it on one line together

Reply:
done